modules/ROOT/pages/_partials/release_notes/sync-gateway-3-2-0-release-note.adoc

```diff
@@ -0,0 +1,215 @@
+== 3.2.0 -- August 2024
+
+[WARNING]
+--
+
+Do not deploy Eventing/Sync Gateway until all SGW nodes are at version 3.2.
+For older Sync Gateway versions that do not write import XATTRs, Eventing functions will encounter infinite recursions and duplicate mutations if deployed in a mixed mode SGW environment.
+This can only happen during a mixed mode of SGW with a 3.2 version and an older version during an upgrade and a new Eventing/Sync Gateway function is deployed.
+
+--
+
+Version 3.2.0 of Sync Gateway delivers the following features and enhancements:
+
+[#maint-3-2-0]
+=== Fixed Issues
+
+* https://jira.issues.couchbase.com/browse/CBG-2905[CBG-2905 -  Fixed repeated attempts to connect to inexistent buckets are done after removing a database and bucket]
```

Comment:
Is it standard to add the word 'Fixed' to most of the fixed issues descriptions as you've done here?  I thought that might be inferred from the fact that these are in the 'Fixed Issues' section.

Reply:
The aim is to make the tickets read more as full sentences on the documentation side